src/sentry/search/events/datasets/sessions.py

```diff
@@ -27,7 +29,34 @@ def field_alias_converter(self) -> Mapping[str, Callable[[str], SelectType]]:
 
     @property
     def function_converter(self) -> Mapping[str, SnQLFunction]:
-        return {}
+        return {
+            function.name: function
+            for function in [
+                SnQLFunction(
+                    "percentage",
+                    required_args=[SessionColumnArg("numerator"), SessionColumnArg("denominator")],
+                    # Since percentage is only used on aggregates, it needs to be an aggregate and not a column
+                    # This is because as a column it will be added to the `WHERE` clause instead of the `HAVING` clause
+                    snql_aggregate=lambda args, alias: Function(
+                        "if",
+                        parameters=[
+                            Function("greater", parameters=[args["denominator"], 0]),
+                            Function("divide", parameters=[args["numerator"], args["denominator"]]),
+                            None,
+                        ],
+                        alias=alias,
+                    ),
+                    default_result_type="percentage",
+                ),
+                SnQLFunction(
+                    "identity",
```

Comment:
Why do you need the `identity` function here?

Reply:
It might have just been a workaround so that we could alias a column to another name now that I'm looking at it